### DISCUSSION TITLE
Should auto-generated list pages show anything out of the box?

### DISCUSSION THREAD

**Discussion by dogweather** [imfing/hextra#255] [Q&A]
I get a heading and that's it. There's no `_index.md` or `index.md` in these directories, but I could add them. Here's the Ananke them vs. Hextra. Am I doing something wrong? What do most people do?

EDIT: I see that `type: blog` will generate a default view. I guess it's by design that `type: docs` does not?

<img width="1624" alt="Screenshot 2024-01-14 at 6 36 15 PM" src="https://github.com/imfing/hextra/assets/150670/0e6b2082-0439-4094-aa2d-425430e12ed6">

<img width="1624" alt="Screenshot 2024-01-14 at 6 36 21 PM" src="https://github.com/imfing/hextra/assets/150670/ac07fc90-4367-46ae-905c-08be3a43fe4f">

<img width="1624" alt="Screenshot 2024-01-14 at 6 37 16 PM" src="https://github.com/imfing/hextra/assets/150670/faf95965-cfee-4ff9-af2c-d4799c2c9ed4">

<img width="1624" alt="Screenshot 2024-01-14 at 6 37 10 PM" src="https://github.com/imfing/hextra/assets/150670/1be889b8-ff58-4c92-a61b-5d4a0a902dd0">

Thanks!

**Comment by imfing** [comment]
> I get a heading and that's it. There's no _index.md or index.md in these directories

I think you need to add those files manually. Did you use `hugo` command to generate those directories?

**Reply by dogweather** [reply]
I'll give it a test with a new Hugo project. 

I'm generating all the content with a separate program that I wrote.
